docker/linksets/.env

```diff
@@ -0,0 +1,9 @@
+#Setting some variable so we can ensure compose works consistently
+#These values dont have to be the same as what is in the dockerfile
+DATABASE_URL=postgis
+S3_SOURCE_DATA_PATH=/source-data
+S3_GEOFABRIC_PATH=/geofabric_2-1/HR_Catchments_GDB_V2_1_1.zip
+ASGS_MB_WFS_URL=https://geo.abs.gov.au/arcgis/services/ASGS2016/MB/MapServer/WFSServer
+S3_LINKSET_PATH=/auto-generated/linksets
```

Comment:
`S3_LINKSET_PATH` should be the version coming from `common.env`. I expect this is for testing as you didn't want to override the manually added data. The value you have is what I want to update `common.env` to. Perhaps you can just make that change now. It will break the cache build, but maybe we can copy the other sets in there (to auto-generated/linksets) for now, to fix that.

Reply:
done but as you say it will break the linksets, I'll create a new issue to fix those